test/integration/connect/test_connect.py

```diff
@@ -37,7 +37,6 @@ def try_execute_on_instance(tt_cmd, tmpdir, instance, file_path, opts=None):
         for k, v in opts.items():
             connect_cmd.append(k)
             connect_cmd.append(v)
-
```

Comment:
Why is this empty line removed?

Reply:
You're right, there's no need. Reverted.